include/fmt/format.h

```diff
@@ -54,7 +54,7 @@
 #include "base.h"
 
 // Checking FMT_CPLUSPLUS for warning suppression in MSVC.
-#if FMT_HAS_INCLUDE(<bit>) && FMT_CPLUSPLUS >= 201703L
+#if FMT_HAS_INCLUDE(<bit>) && FMT_CPLUSPLUS >= 202002L
```

Comment:
I suggest replace the ``>=`` with ``>`` for compatibility with gcc-9, which contains ``<bit>`` but ``FMT_CPLUSPLUS`` is less than ``202002L``.